### PR TITLE
Tidy PR request

### PR DESCRIPTION
**Pull request by Abalalojik**

### CODE REVIEW

File: ark-net/Utils/Enum/EnumExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace ArkTribeLibrary
```
[the remark this answers is not in the document]
Comment (Abalalojik) [reply ark-net/Utils/Enum/EnumExtensions.cs]: I'm going to remove it.

[the remark this answers is not in the document]
Comment (Abalalojik) [reply ark-net/Utils/Enum/EnumExtensions.cs]: This method was to provide easy way to put the enum's content into a lable, for example to show what the state is (mainnet / devnet, or the language and so on by using an attribute [description("desc")] above the enumeration.

**Comment by Abalalojik** [comment]
Okay. the changes are done. It indeed automatically added my changes.
Can you tell me if it's okay?

**Comment by kristjank** [comment]
Ok Merged. 

@Abalalojik next time we do this in small batches

**Pull request merged**